### ISSUE TITLE
RNN for Stock Price Prediction

### ISSUE THREAD

**Issue by Ratz21** [Ratz21/LPV#10]
import pandas as pd
import numpy as np
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import LSTM, Dense
from sklearn.preprocessing import MinMaxScaler

data = pd.read_csv('GOOG.csv')['Close'].values.reshape(-1, 1)
scaler = MinMaxScaler()
scaled = scaler.fit_transform(data)

X, y = [], []
for i in range(60, len(scaled)):
    X.append(scaled[i-60:i])
    y.append(scaled[i])
X, y = np.array(X), np.array(y)

model = Sequential([
    LSTM(50, return_sequences=True, input_shape=(X.shape[1], 1)),
    LSTM(50),
    Dense(1)
])
model.compile(optimizer='adam', loss='mse')
model.fit(X, y, epochs=10)
